apps/taxes/src/modules/avatax/ui/edit-avatax-configuration.tsx

```diff
@@ -4,13 +4,16 @@ import { useRouter } from "next/router";
 import React from "react";
 import { z } from "zod";
 import { trpcClient } from "../../trpc/trpc-client";
-import { AvataxConnectionObfuscator } from "../avatax-connection-obfuscator";
-import { AvataxConfig } from "../avatax-connection-schema";
+import { AvataxObfuscator } from "../avatax-obfuscator";
+import { AvataxConfig, BaseAvataxConfig } from "../avatax-connection-schema";
 import { AvataxConfigurationForm } from "./avatax-configuration-form";
+import { useAvataxConfigurationStatus } from "./configuration-status";
 
-const avataxObfuscator = new AvataxConnectionObfuscator();
+const avataxObfuscator = new AvataxObfuscator();
```

Comment:
I think this approach is a rabbit hole and unnecessary overhead. Im not saying to drop it but IMO we can just return token to the frontend IF user has `manage apps` permission